src/common/wallet-state.ts

```diff
@@ -529,6 +537,31 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
     web3.eth.sendSignedTransaction(tx.rawTransaction) // ETCM-134
   }
 
+  const doTransfer = async (
```

Comment:
doTransfer?

Reply:
Can we think of a better name?

Reply:
What's wrong with this one? IMHO it describes exactly the thing that's hapenning here